requirements.txt

```diff
@@ -1,5 +1,5 @@
 alembic==1.4.2
-canonicalwebteam.flask_base==0.6.0
+git+git://github.com/canonical-web-and-design/canonicalwebteam.flask-base.git@favicon_root#egg=canonicalwebteam.flask_base
```

Comment:
Should this be canonicalwebteam.flask_base==0.6.1 instead?

Reply:
Seems to be the cause of the failures:

#12 3.807   ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git clone -q git://github.com/canonical-web-and-design/canonicalwebteam.flask-base.git /tmp/pip-install-p72ioa8i/canonicalwebteam.flask-base
#12 3.818 ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?

ERROR: Command errored out with exit status 128: git clone -q git://github.com/canonical-web-and-design/canonicalwebteam.flask-base.git /tmp/pip-install-z0dntiyp/canonicalwebteam.flask-base Check the logs for full command output.

Reply:
Yes, but I use this to test before releasing to Pypi and swap it after I publish.